docs/apps/address-book.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+sidebar_position: 6
```

Comment:
Not something to necessarily change right now, but I strongly recommend at least two-digit sidebar position numbers (10, 20, 30) to start. That way, you can insert items later (i.e. 25) without renumbering.